heron/tools/common/src/python/utils/config.py

```diff
@@ -95,6 +97,7 @@ def get_subparser(parser, command):
   for subparsers_action in subparsers_actions:
     # get all subparsers
     for choice, subparser in subparsers_action.choices.items():
+      print choice, subparser
```

Comment:
is this `print` expected behavior?

Reply:
no left over from debugging. will remove